pages/linux/toolbox-create.md

```diff
@@ -0,0 +1,16 @@
+# toolbox create
+
+> Create a new `toolbox` container.
+> More information: <https://www.mankier.com/1/toolbox-create>.
+
+- Create a `toolbox` container for a different distribution other than the host:
+
+`toolbox create --distro {{distribution}}`
```

Comment:
How do you specify the container's name?

Reply:
We specify it in the place of {{distribution}}. Example: `toolbox create --distro fedora` or if your host is fedora then you can directly use `toolbox create` alone.